addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -659,6 +663,8 @@ def create(self, vals_list):
         for values in vals_list:
             if 'state' in values and values['state'] not in ('draft', 'confirm'):
                 raise UserError(_('Incorrect state for new allocation'))
+            if len(values.get('employee_ids', [])) > 1 :
```

Comment:
Value could be ``[(5, 0, 0), (4, 1)]`` and this won't be correct in that case (even if it doesn't make a lot of sense to set that value on a create). Manipulating X2Many values is usually a bad idea. Couldn't you update the field after creation ?